xq/__main__.py

```diff
@@ -12,23 +12,41 @@
 from . import NAME, DESCRIPTION, VERSION
 
 
-def wrap_in_results(elements: [Union[etree._Element, etree._ElementUnicodeResult]]) -> etree._Element:
+def wrap_in_results(
+    elements: [Union[etree._Element, etree._ElementUnicodeResult]]
+) -> etree._Element:
     results = E.results()
     for el in elements:
         results.append(E.result(el))
     return results
 
 
-def apply_xpath(infile, xpath_query=None, colorize=False):
+def raw_out(
+    elements: [Union[etree._Element, etree._ElementUnicodeResult]]
+) -> etree._Element:
+    def tostr(e):
+        if type(e) is etree._ElementUnicodeResult:
+            return str(e)
+        if type(e) is etree._Element:
+            return '[Element <%s>]' % e.tag
+        raise ValueError('This should never happen: unsupported type of match')
+    results = [tostr(e) for e in elements]
+    return results
+
+
+def apply_xpath(infile, xpath_query=None, colorize=False, raw=False):
     try:
         parsed = etree.parse(infile, etree.XMLParser(remove_blank_text=True))
     except etree.XMLSyntaxError:
         parsed = etree.parse(infile, etree.HTMLParser(remove_blank_text=True))
 
     if xpath_query:
         matches = parsed.xpath(xpath_query)
-        results = wrap_in_results(matches)
-        output = etree.tostring(results, pretty_print=True)
+        if not raw:
+            results = wrap_in_results(matches)
+            output = etree.tostring(results, pretty_print=True)
+        else:
+            output = "\n".join(raw_out(matches))
```

Comment:
Just an inversion of "if" test
```suggestion
        if raw:
            output = "\n".join(raw_out(matches))
        else:
            results = wrap_in_results(matches)
            output = etree.tostring(results, pretty_print=True)
```